docs/troubleshooting.md

```diff
@@ -16,3 +16,32 @@ Figure 1: Troubleshooting overview diagram
 ## Common Errors
 ### Windows log with Error code 2150859027
 If you are on Windows 2016 or higher and are getting Error code 2150859027, or messages about HTTP URLs not being available in your Windows logs, we suggest looking at [this guide.](https://support.microsoft.com/en-in/help/4494462/events-not-forwarded-if-the-collector-runs-windows-server-2019-or-2016)
+
+### No logs forwarded from member servers
+Check the following:
+
+* Sysmon service is running on the client
+* The [LME-WEC-Client-GPO](https://github.com/ukncsc/lme/blob/master/Chapter%201%20Files/lme_gpo_for_windows.zip) is applying to the member server
+* That the member server has been rebooted to apply permissions to the logs ([see issue #41](https://github.com/ukncsc/lme/issues/41#issuecomment-554037796))
+
+### Events not forwarding from Domain Controllers
+Please be aware that Logging Made Easy does not currently support logging Domain Controllers, and the log volumes may be significant from servers with this role.  If you wish to proceed forwarding logs from your Domain Controllers please be aware you do this at your own risk!  Monitoring such servers has not been tested or endorsed by the NCSC and may have unintended side effects.
+
+If you wish to monitor Domain Controllers then the contents of [this blog post](https://rockyprogress.wordpress.com/2011/12/04/security-event-log-collection-from-a-domain-controller/) may be helpful.  Note the NCSC is not responsible for the content of external sites.
```

Comment:
I would be more comfortable if this section (line 30) is removed. I think linking to 10 year old blog that we have no control over is not advised, and gives mixed messages.... "It's not supported, but look here" is not consistent. etc

Reply:
Hi @a-d-a-m-b, I've removed L30 - L33 as the reminder seemed superfluous given it was right next to the warning.  Valid points on referencing 10 year old content.